src/interfaces/store-interfaces/state-interfaces/name-request-interface.ts

```diff
@@ -20,6 +22,8 @@ export interface NameRequestDetailsIF {
 
 // Name request applicant details interface
 export interface NameRequestApplicantIF {
+  address?: string
```

Comment:
Is address really a string?

Reply:
I will rename the prop name, in this case it is sort formatted address string. A combination of all the individual fields for display purposes.
So possibly a better name for the prop would be `fullAddress` or `fullDisplayAddress` something along those lines.

Reply:
Updated in next commit

Reply:
That would tie in nicely with the common mixin method `formatFullAddress()` :)

Reply:
Definitely :)